src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -5380,7 +5381,7 @@ void Assembler::evpmovzxbw(XMMRegister dst, KRegister mask, Address src, int vec
   assert(dst != xnoreg, "sanity");
   InstructionMark im(this);
   InstructionAttr attributes(vector_len, /* rex_w */ false, /* legacy_mode */ _legacy_mode_bw, /* no_mask_reg */ false, /* uses_vl */ true);
-  attributes.set_address_attributes(/* tuple_type */ EVEX_HVM, /* input_size_in_bits */ EVEX_NObit);
+  attributes.set_address_attributes(/* tuple_type */ EVEX_QVM, /* input_size_in_bits */ EVEX_NObit);
```

Comment:
This should have remained as EVEX_HVM.

Reply:
https://www.felixcloutier.com/x86/pmovzx#:~:text=VPMOVZXBW%20zmm1%20%7Bk1%7D%7Bz%7D%2C%20ymm2/m256-,B,-V/V

https://www.felixcloutier.com/x86/pmovzx#:~:text=N/A-,B,Half%20Mem,-ModRM%3Areg%20(w